src/main/java/ru/itis/services/WishListService.java

```diff
@@ -21,13 +21,14 @@ public WishListService(WishListRepository wishListRepository, UserService userSe
     }
 
     public WishList addNewWishList(String title, String token) {
-        User currentUser = userService.findUserByToken(token).get();
-        WishList newWL = WishList.builder()
-                .title(title)
-                .author(currentUser)
-                .build();
-        currentUser.getWishLists().add(newWL);
-        return wishListRepository.save(newWL);
+        if (userService.findUserByToken(token).isPresent()) {
```

Comment:
Rewrite this method using guard expression.
```java
if (!user.present()) { return defaultWishList(); }
new WishList();
// logic
```

Reply:
fixed